src/main/java/com/jalasoft/jfc/model/strategy/CommandVideoConverter.java

```diff
@@ -29,6 +29,13 @@ public CommandVideoConverter(String commandValue) {
         this.commandValue = commandValue;
     }
 
+    /**
+     * Creates a new CommandVideoConverter object.
+     * constructor empty.
+     */
+    public CommandVideoConverter() {
```

Comment:
if this costructor is empty, is not necessary implement it, java adds a default empty constructor